chia/daemon/server.py

```diff
@@ -1102,30 +1102,40 @@ async def stop_service(self, request: Dict[str, Any]) -> Dict[str, Any]:
         return response
 
     async def is_running(self, request: Dict[str, Any]) -> Dict[str, Any]:
-        service_name = request["service"]
-
-        if service_name == service_plotter:
-            processes = self.services.get(service_name)
-            is_running = processes is not None and len(processes) > 0
-            response = {
-                "success": True,
-                "service_name": service_name,
-                "is_running": is_running,
-            }
+        service = request.get("service", "*")
```

Comment:
What if this were a list of strings instead and we didn't have to apply nested parsing to the existing JSON parsed result?  Or perhaps `service` remains entirely as is, there's a new key named `services` that's a list, we just prepend `service` to `services` and move forward with only looking at `services`.  I don't think I've quite thought through all the little corners.

Reply:
@xdustinface and I have been discussing the merits of just creating a new `running_services` command rather than trying to shoehorn this new behavior into `is_running`. I was initially going for something small and quick, but I'm onboard with leaving `is_running` as-is and moving this new functionality to a new command.

Reply:
Added `running_services` instead of modifying `is_running`